demo_umami_content/src/InstallHelper.php

```diff
@@ -244,6 +245,40 @@ public function importPages() {
     return $this;
   }
 
+  /**
+   * Imports block contents.
+   *
+   * @return $this
+   */
+  public function importBlockContent() {
+    $block_content_metadata = [
+      'umami_recipes_banner' => [
+        'uuid' => '4c7d58a3-a45d-412d-9068-259c57e40541',
+        'info' => 'Umami Recipes Banner',
+        'type' => 'banner_block',
```

Comment:
We should be able to add a simple assert to the existing test that asserts this is created, removed and then recreated.

Reply:
I will add to the tests.